stream_alert_cli/helpers.py

```diff
@@ -125,6 +127,9 @@ def tf_runner(**kwargs):
     if action == 'destroy':
         tf_command.append('-destroy')
 
+    if not refresh_state:
```

Comment:
This logic should switch per the previous comment

Reply:
how should it switch? what would it do in case `refresh_state` is True?

Reply:
Are you thinking of something like that?
` if refresh_state:
    tf_command.append('-refresh=true')
  else:
        tf_command.append('-refresh=false') `

Reply:
actually, this doesn't need to change.

Reply:
That's what I thought too :smile: